src/main/java/org/mskcc/cbio/oncokb/web/rest/SlackController.java

```diff
@@ -125,11 +125,21 @@ public ResponseEntity<String> approveUser(@RequestParam("payload") String action
                             mailService.sendApiAccessApprovalEmail(userDTO);
                         }
                         break;
-                    case GIVE_TRIAL_ACCESS:
+                    }
+                    case GIVE_TRIAL_ACCESS: {
```

Comment:
We don't use braces in switch.

Reply:
I added these so I could also define `updatedUser` here even though it was defined in the previous statement (since the cases all share the same scope). I can change the name and remove the braces, however.